plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -849,6 +850,11 @@ public JdbcOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
         }
     }
 
+    protected void rollbackCreateDestinationTable(ConnectorSession session, JdbcOutputTableHandle handle)
+    {
+        dropTable(session, handle.getRemoteTableName(), false);
```

Comment:
Why do we want to take `JdbcOutputTableHandle` instead of `RemoteTableName` as the method parameter?